interfaces/s3/v0/schemas/provider.json

```diff
@@ -98,6 +98,19 @@
                 ]
             ]
         },
+        "service-account": {
+            "title": "Service account",
+            "description": "The service account file content, in base64",
+            "type": "string",
+            "items": {
+                "type": "string"
+            },
+            "examples": [
+                [
+                    "base64-encoded-ca-chain=="
```

Comment:
This doesn't look like a good example for a service account

Reply:
Hi @gruyaume This PR has been stopped as we are not yet clear if we will use the s3-integrator for GCS integration. I will move it to a draft state.